db/migrate/20230607203832_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] From the ERD, I could see that the user model was referenced as author_id as a relationship with other models. So to ensure that your schema matches the ERD, it will be better to use author_id to reference users.
e.g

In your likes migration file you should have  something like :
`t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index `